charts/osdfir-infrastructure/README.md

```diff
@@ -240,6 +241,27 @@ kubectl delete pvc -l release=osdfir-release
 | `turbinia.redis.replica.resources.limits`   | The resources limits for the Redis replica containers                                        | `{}`             |
```

Comment:
maybe give an example of what to set this value to.

Reply:
I think if we do this we should set examples for the rest of the `resources` values (for all the Turbinia/TS pods and 3rd party dependencies). If a user was looking at the `values.yaml` file they would see an example there of how it's set, but maybe we can revisit this in another bug to also provide examples that are rendered in the README. 

Thinking we can have two options, either in each description put something like `The resources limits for the Redis replica containers (e.g. limits: cpu: Xm memory XGi)` or instead just set default values. I didn't set default values at first so that whoever is deploying the chart can run it on the most minimal resources e.g. minikube but maybe instead we can consider just having very small default values and adding a resource requirements at the top

Reply:
Let's do this in another PR. I am good with either but as these are a lot of settings it would be cool if we can make the absolute minimum default so people can deploy locally through eg minikube?